docs/index.html

```diff
@@ -2,11 +2,22 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>captains-log - List out release notes in a digestable way</title>
+  <title>A Drone plugin that provides automated release notes.</title>
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-  <meta name="description" content="List out release notes in a digestable way">
+  <meta name="description" content="A Drone plugin that provides automated release notes.">
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css">
+
+  <!-- third-generation iPad with high-resolution Retina display: -->
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="./static/favicon.png">
```

Comment:
Are relative paths correct here?

Reply:
They worked locally, I'm unsure if they will work at the doc level. However, Docsify leads me to believe that if it runs locally, it "should" work in a prod environment.